enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -64,8 +64,11 @@ var (
 
 	// TODO(iain): delete executor.podman.run_soci_snapshotter flag once
 	// the snapshotter doesn't need root permissions to run.
-	sociStoreLogLevel            = flag.String("executor.podman.soci_store_log_level", "", "The level at which the soci-store should log. Should be one of the standard log levels, all lowercase.")
-	runSociStoreLocally          = flag.Bool("executor.podman.run_soci_store", true, "If true, runs the soci store locally if needed for image streaming.")
+	sociStoreLogLevel   = flag.String("executor.podman.soci_store_log_level", "", "The level at which the soci-store should log. Should be one of the standard log levels, all lowercase.")
+	runSociStoreLocally = flag.Bool("executor.podman.run_soci_store", true, "If true, runs the soci store locally if needed for image streaming.")
+
+	// TODO(https://github.com/buildbuddy-io/buildbuddy-internal/issues/2570): remove this flag
+	runSociStoreRaceDetector     = flag.Bool("executor.podman.dev_only_soci_store_race_detector", false, "If true, runs the soci store with race detection enabled (up to 10x slower). Only for us in dev.")
```

Comment:
instead of adding a separate flag, what do you think about adding a flag for the path to the soci binary?

Reply:
Sounds good to me, I went ahead and combined it with the run-locally flag.